juniper_official/Chassis/netsvc.rule

```diff
@@ -0,0 +1,37 @@
+iceberg {
+    topic chassis.networkservices {
+        rule netsvc-rule {
+            sensor netsvc-sensor {
+                iAgent {
+                    file netsvc.yml;
+                    table NetSvcTable;
+                    frequency 60s;
+                }
+            }
+            trigger netsvc-sync {
+                term synched {
```

Comment:
Default trigger frequency is 10s. Do we want to execute the trigger every 10s when sensor frequency is 60s?

Reply:
How would a user know what the default is? 

root@20a6061bff84# set iceberg topic chassis.networkservices rule netsvc-rule trigger netsvc-sync frequency ?
Possible completions:
  <frequency>          Frequency or time interval at which the trigger needs to be evaluated. Specify positive integer followed by s/m/h/d/w/y representing seconds/minutes/hours/days/weeks/years. Eg: 2s
[edit]

What is best practice? Should it be the same as sensor frequency? 60s is probably too low for iAgent sensor. I'll discuss with my team.